site/content/en/docs/Installation/Creating Cluster/minikube.md

```diff
@@ -29,9 +29,16 @@ minikube profile agones
 The following command starts a local minikube cluster via virtualbox - but this can be
 replaced by a [vm-driver](https://github.com/kubernetes/minikube#requirements) of your choice.
 
+{{% feature publishVersion="1.9.0" %}}
```

Comment:
This one should be expiryVersion

Reply:
Sorry, my fault, just thought that could be the case, while preparing another PR.